### DISCUSSION TITLE
Cannot Tab to Link Component

### DISCUSSION THREAD

**Discussion by jrock2004** [vercel/next.js#40228] [Help]
I am build a nav that uses the link component but when I render it in the browser, I cannot tab to those links. Is there special code I need to make it work?

```tsx
<Link className="outlink-pink-700" href={{ pathname: path }}>
  {title}
</Link>
```

The paths are internal paths. Example of what I pass in via `path` is `/services`

**Comment by icyJoseph** [comment]
- What browser are you using? https://www.scottohara.me/blog/2014/10/03/link-tabbing-firefox-osx.html
- Do you observe the same behaviour with regular anchor tags `<a></a>`?

**Reply by jrock2004** [reply]
Firefox. I do not have any a tags since there is the link component. I will try though

**Reply by icyJoseph** [reply]
The next/link component renders an anchor tag. 

However if you use custom components, or do not have a string as direct children to the Link component, then yeah, there could be problems.

The reason to use it is to let the router know about navigation. As far as the generated HTML is concerned there's an anchor tag there, if used correctly.

**Reply by icyJoseph** [reply]
Look at this example from the docs, https://nextjs.org/docs/api-reference/next/link:

```jsx
import Link from 'next/link'

function Home() {
  return (
    <ul>
      <li>
        <Link href="/">
          <a>Home</a>
        </Link>
      </li>
      <li>
        <Link href="/about">
          <a>About Us</a>
        </Link>
      </li>
      <li>
        <Link href="/blog/hello-world">
          <a>Blog Post</a>
        </Link>
      </li>
    </ul>
  )
}
```

The Link wraps an anchor tag. 

However if the Link wraps a string, Next.js automatically wraps with an anchor tag. There's some ongoing migration, https://github.com/vercel/next.js/pull/36436, that's somewhat related to that, but the string wrapping has been there for quite a while now.

If the DOM has no anchor tags, then you are most definitely doing something wrong, but the way you have it with only a string `{title}` should work.

What next version are you using?
